### ISSUE TITLE
Improve Web Accessibility

### ISSUE THREAD

**Issue by gtirloni** [mattermost/platform#2452]
While suggesting the use of Mattermost as a replacement for Slack/IRC, it was mentioned to me that it would pose a problem for our target audience due to some accessibility issues. Right now, Slack isn't very great in that area either.

I'm opening this issue so the [WCAG 2.0](https://en.wikipedia.org/wiki/Web_Content_Accessibility_Guidelines) guidelines may be considered in the future.

**Comment by gtirloni** [comment]
Link to feature idea forum: https://mattermost.uservoice.com/forums/306457-general/suggestions/13036146-improve-web-accessibility
